src/eterna/AnnotationManager.ts

```diff
@@ -427,7 +409,10 @@ export default class AnnotationManager {
             ) {
                 this._selectedAnnotation = null;
             }
-            this.onTriggerPoseUpdate.emit();
+            this.updateAnnotationViews();
```

Comment:
Was there a motivation for favoring this here as opposed to using `this.propagateDataUpdates()` like you did elsewhere?

Reply:
I went back and forth on it, and decided on this because it only affects visual state, not persisted state